src/components/a-checkbox/index.js

```diff
@@ -7,6 +8,14 @@ import template from './_template';
 
 class AXACheckbox extends BaseComponentGlobal {
   static tagName = 'axa-checkbox'
+  static propTypes = {
+    inputId: PropTypes.string,
+    error: PropTypes.bool,
+    value: PropTypes.bool,
+    name: PropTypes.string,
+    checked: PropTypes.bool,
+    disabled: PropTypes.bool,
+  }
 
   static get observedAttributes() {
```

Comment:
The returned array from `get observedAttributes` could be derived now from propTypes 💪

Reply:
nice! love it

Reply:
Okay I gave it a first try, still need to test

Reply:
works yeah 😃